docs/content/management-bundles.md

```diff
@@ -252,6 +252,189 @@ and number keys are converted to strings. Also, YAML !!binary tags are not
 supported.
 {{< /info >}}
 
+### Bundle Container Format
+
+OPA is able to interact with [OCI](https://opencontainers.org/) compatible registries to be able to download and use policies stored as containers.
+To configure OPA to use an OCI repository see the [service configuration section](../configuration/#services)
+
+**Structure**
+The bundle container is composed of 3 layers:
+- the manifest layer - contains the information about the tarball layer of the container(the digest, size, mediatype and annotations) and the config layer
+- the bundle tarball layer - the actual bundle tarball
+- the configuration layer - currently empty
+
+For OCI compatible registries an ***oci*** folder is created in the [persistence directory](../configuration/#miscellaneous). If this value is not set, because the OCI downloader plugin requires a storage path, the system's temporary folder location will be used instead. This folder should be maintained by the user. We recommend backing-up or cleaning up this folder periodically as this acts as a local cache for the OCI downloader. 
+
+**Current Limitations**
+The OCI Downloader plugin used by OPA has a couple of limitation:
+- it accepts only **one** layer per image that contains the bundle tarball
+- it can download only the following application media types: 
+    - `application/vnd.oci.image.layer.v1.tar+gzip`
+    - `application/vnd.oci.image.manifest.v1+json`
+    - `application/vnd.oci.image.config.v1+json`
+
+#### Building and Publishing Policy Containers
+
+There are multiple ways to build an image from a policy code base using different tools.
+
+#### Using policy CLI
+
+The [policy CLI](https://www.openpolicyregistry.io/docs/cli/download) tool can be easily used to build and push a policy bundle to a remote OCI registry using just two simple commands:
```

Comment:
Let's keep OPR and the policy CLI out of it, please. It's vendor-provided tooling, we don't want to link to that. Please refer to the OPA docs in the OPR docs, but not this way around.

Reply:
The PR was updated recently to address this concern. I'm marking this as resolved. :+1: